scripts/install.sh

```diff
@@ -1,10 +1,19 @@
 #!/bin/bash --login
-#
+
+cwd=$(pwd)
+
+if [[ $cwd != *"topological_inventory-guides/scripts" ]]; then
```

Comment:
@lpichler can you try if this condition is allowed on MAC?

Reply:
yes it is 👍 

when you are using `pwd` from symlink  - that the condition is false